### DISCUSSION TITLE
Framework a utilizar

### DISCUSSION THREAD

**Discussion by Yoel-Gasca** [Launch-X-Latam/MisionFrontEnd#31] [Polls]
Encuesta:

**Comment by Cain-ARG** [comment]
Mientras que no sea React, cualquiera lo que pasa con React a mi parecer es que esta super popular pero esta lleno linkedin de Junior con JS y React, saturan un poco ese mercado.

**Reply by rfponce** [reply]
Aunque esté saturado hay mucha demanda de ese framework. Haz una búsqueda en Linkedin Empleos y verás.

**Reply by SebasMarv** [reply]
Concuerdo, la mayoría de empleos desean como skill para un perfil de frontend el framework React, y muy pocas desearian hacer el cambio a otro framework.

**Comment by magouwuxd** [comment]
React ya lo se, pero estaria chido que lo pudiera poner en mi CV que tome curso por parte de microsoft de React, a ver si asi los headhunters me toman en cuenta :(, aunque me gustaria aprender Angular

**Comment by rgplazas** [comment]
Gracias

**Comment by Dhinestroza05** [comment]
Svelte js

**Comment by NestorMyrie** [comment]
React! estaria genial

**Comment by gersonjeanpierre** [comment]
La gente lo aclama! React!

**Comment by GeovannyCh** [comment]
Como lo veo, cualquiera de los 3 sirven, ya por cuestión de gustos o necesidades, deberemos aprender algún otro framework mas adelante en nuestra carrera profesional.

**Reply by MarTorres07** [reply]
Pense que en el evento de inicio de launch x dijeron que ibamos a ver Angular? que mal que no sea angular :/

**Reply by GeovannyCh** [reply]
Por lo que e leído, lo que pasa con Angular es que tiene una curva de aprendizaje mas alta que los otros 2, teniendo en cuenta eso y la duración del launch x, lo mas probable es que se nos enseñe react o vue.

**Comment by RafaelMartinCallataFlores** [comment]
veo en las publicaciones de LinkedIn que piden React para un puesto y estaría bueno poder aprender en este curso
